src/libraries/Microsoft.VisualBasic.Core/tests/Microsoft/VisualBasic/FileIO/FileSystemTests.cs

```diff
@@ -847,6 +847,27 @@ public void RenameFile_File_NewName()
         //   public void WriteAllText(string file, string text, bool append) { }
         //   public void WriteAllText(string file, string text, bool append, System.Text.Encoding encoding) { }
 
+        /// <summary>
```

Comment:
This PR currently only contains a new test.  Did you intend to include a src fix as well?

Reply:
Yes, I wanted to verify the test hit the expected failures on osx and linux before making any src changes. I'll work on a fix next.